src/routes/+layout.svelte

```diff
@@ -26,37 +26,38 @@
         <AppBar>Hazy Goods</AppBar>
     </svelte:fragment>
 
-    <AppRail>
-        <svelte:fragment slot="lead">
-            <AppRailAnchor href="/">
-                <div class="ml-7">
-                    <Icon
-                        src={BuildingStorefront}
-                        size="20px"
-                        theme="outline"
-                    />
-                </div>
-                Home</AppRailAnchor
-            >
-            <AppRailAnchor href="/dispensary">
-                <div class="ml-7">
-                    <Icon
-                        src={ClipboardDocumentList}
-                        size="20px"
-                        theme="outline"
-                    />
-                </div>
-                Dispensary List</AppRailAnchor
-            >
-            <AppRailAnchor href="/dispensary/add">
-                <div class="ml-7">
-                    <Icon src={PlusCircle} size="20px" theme="outline" />
-                </div>
-                Add New Dispensary</AppRailAnchor
-            >
-        </svelte:fragment>
-    </AppRail>
-    <main>
```

Comment:
When I inspected the DOM, it looks like the AppShell already applies a `<main>` tag, so no need to have another. (For the record this is not criticism, I think I was the one to do this in the first place, just noting the change)